### PR TITLE
Jakarta Connectors 2.1

### PR DESCRIPTION
**Pull request by smillidge**
Signed-off-by: Steve Millidge <steve.millidge@payara.fish>

## Specification PR template
When creating a specification project release review, create PRs with the content defined as follows.

Include the following in the PR:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version, and the directory contains the following.
- [x] Specification PDF in the form of jakarta-wombat-spec-x.y.pdf
- [x] Specification HTML in the form of jakarta-wombat-spec-x.y.html
- [x] A specification page named _index.md following the template at:
      https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md
- [x] For a Progress Review, that sufficient progress has been made on a Compatible Implementation and TCK, to ensure that the spec is implementable and testable.
- [x] For a [Release Review](https://www.eclipse.org/projects/handbook/#release-review), a summary that a Compatible Implementation is complete, passes the TCK, and that the TCK includes sufficient coverage of the specification. The TCK users guide MUST include the instructions to run the compatible implementations used to validate the release.
Instructions MAY be by reference.
   - [x] Updated release record
   - [x] Generated IP Log
   - [x] Email to PMC
   - [x] Start release review by emailing EMO 
- [x] The URL of the OSSRH staging repository for the api, javadoc:
      https://jakarta.oss.sonatype.org/service/local/repo_groups/staging/content/jakarta/resource/jakarta.resource-api/2.1.0/jakarta.resource-api-2.1.0.jar 
      https://jakarta.oss.sonatype.org/service/local/repo_groups/staging/content/jakarta/resource/jakarta.resource-api/2.1.0/jakarta.resource-api-2.1.0-javadoc.jar
- [x] The URL of the staging directory on downloads.eclipse.org for the proposed EFTL TCK binary:
     https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee10/staged/eftl/jakarta-connectors-tck-2.1.0.zip
- [x] The URL of the compatibility certification request issue:
      https://github.com/eclipse-ee4j/jca-api/issues/132
- [x] Specification JavaDoc in the wombat/x.y/apidocs directory. 
If desired, an optional second PR can be created to contain just the JavaDoc in the `apidocs` directory.

Note: If any item does not apply, check it and mark N/A below it.

### CODE REVIEW

**Comment by scottmarlow** [comment]
Connectors 2.1 TCK to try would be either https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee10/staged/eftl/jakarta-connectors-tck-2.1.0.zip

**Comment by ivargrimstad** [comment]
The Copyright date must be updated to 2018, 2022 in the spec document
Status, and release fields in the PDF and HTML missing

File: connectors/2.1/_index.md [hunk 1]
```diff
@@ -1,11 +1,43 @@
...
+  * [jakarta.resource:jakarta.resource-api:jar:2.1.0](https://search.maven.org/artifact/jakarta.resource/jakarta.resource-api/2.1.0/jar)
+* Compatible Implementations used for [ratification](https://www.eclipse.org/projects/efsp/?version=1.2#efsp-ratification).
+  * [Eclipse Vombatidae 9.1.1](https://github.com/eclipse-ee4j/vombatidae)
+
+# Compatible Implementations
+
+
+# Ballots
+
+## Release Review
```
[the remark this answers is not in the document]
Comment (smillidge) [reply connectors/2.1/_index.md:25]: Done

File: connectors/2.1/apidocs/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Manifest-Version: 1.0
```
[the remark this answers is not in the document]
Comment (smillidge) [reply connectors/2.1/apidocs/META-INF/MANIFEST.MF]: Removed

File: connectors/2.1/_index.md [hunk 3]
```diff
@@ -1,11 +1,43 @@
...
+* [Jakarta Connectors 2.1 Release Record](https://projects.eclipse.org/projects/ee4j.wombat/releases/1.0)
+* [Jakarta Connectors 2.1 Specification Document](./jakarta-connectors-spec-2.1.pdf) (PDF)
+* [Jakarta Connectors 2.1 Specification Document](./jakarta-connectors-spec-2.1.html) (HTML)
+* [Jakarta Connectors 2.1 Javadoc](./apidocs)
+* [Jakarta Connectors 2.1 XML Schema](https://jakarta.ee/xml/ns/jakartaee/connector_2_0.xsd)
+* [Jakarta Connectors 2.1TCK](http://downloads.eclipse.org/jakarta/wombat/1.0.0/wombat-tck-1.0.0.zip)
+* Maven coordinates
+  * [jakarta.resource:jakarta.resource-api:jar:2.1.0](https://search.maven.org/artifact/jakarta.resource/jakarta.resource-api/2.1.0/jar)
+* Compatible Implementations used for [ratification](https://www.eclipse.org/projects/efsp/?version=1.2#efsp-ratification).
+  * [Eclipse Vombatidae 9.1.1](https://github.com/eclipse-ee4j/vombatidae)
```
[the remark this answers is not in the document]
Comment (smillidge) [reply connectors/2.1/_index.md]: Done although final exact version needs to be added

**Comment by smillidge** [comment]
TCK link updated to the one suggested by Scott

**Comment by smillidge** [comment]
Updated copyright and status in the specification documents

File: connectors/2.1/_index.md [hunk 4]
```diff
@@ -1,11 +1,31 @@
...
 ---
 The Jakarta Connectors specification defines a standard architecture for Jakarta EE application components to connect to Enterprise Information Systems.
 .
 
+* [Jakarta Connectors 2.1 Release Record](https://projects.eclipse.org/projects/ee4j.jca/releases/2.1.0)
+* [Jakarta Connectors 2.1 Specification Document](./jakarta-connectors-spec-2.1.pdf) (PDF)
+* [Jakarta Connectors 2.1 Specification Document](./jakarta-connectors-spec-2.1.html) (HTML)
+* [Jakarta Connectors 2.1 Javadoc](./apidocs)
+* [Jakarta Connectors 2.1 XML Schema](https://jakarta.ee/xml/ns/jakartaee/connector_2_0.xsd)
+* [Jakarta Connectors 2.1 TCK](https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee10/staged/eftl/jakarta-connectors-tck-2.1.0.zip)
```
Comment (ivargrimstad) [inline connectors/2.1/_index.md:14]: This link must point to the final location the TCK will get after being promoted by the spec committee

https://download.eclipse.org/jakartaee/connectors/2.1//jakarta-connectors-tck-2.1.0.zip

Comment (ivargrimstad) [reply connectors/2.1/_index.md:14]: Also add links to sig, sha, and pub like this:

([sig](https://download.eclipse.org/jakartaee/connectors/2.1/jakarta-connectors-tck-2.0.0.zip.sig),  [sha](https://download.eclipse.org/jakartaee/connectors/2.1/jakarta-connectors-tck-2.0.0.zip.sha256),  [pub](https://jakarta.ee/specifications/jakartaee-spec-committee.pub))

Comment (smillidge) [reply connectors/2.1/_index.md]: OK done

**Comment by smillidge** [comment]
PR to update the API POM https://github.com/eclipse-ee4j/jakartaee-api/pull/119/files

**Comment by ivargrimstad** [comment]
As soon as https://github.com/eclipse-ee4j/glassfish/pull/23902 is merges, this is all ready for ballot. 

@smillidge Please send an email to the EMO and PMC, so we can tick all boxes

**Comment by smillidge** [comment]
@ivargrimstad what do I put in the email? Do I need to do an IP Log as this is a minor?

**Comment by ivargrimstad** [comment]
> 

All of the steps can be done from the project page: https://projects.eclipse.org/projects/ee4j.jca 

For the EMO review: Click on schedule a release review and follow the instructions
For the PMC: click on "Email the PMC" (right menu), or just send an email requesting approval manually
Regarding the IP log, good question. I'd just create it to be secure. It's a one-click operation from the project page (right menu)

**Comment by smillidge** [comment]
Emailed PMC, Scheduled the release review and generated the IP Log.
Not sure if I have to do more for the release review i.e. email EMO

**Comment by ivargrimstad** [comment]
> Emailed PMC, Scheduled the release review and generated the IP Log. Not sure if I have to do more for the release review i.e. email EMO

Looks good!
I'll go quickly over once more, and start the ballot shortly

**Review by ivargrimstad — APPROVED** [review]

**Comment by ivargrimstad** [comment]
# On ballot completion, the specification committee mentor:
 - [ ] adds this final checklist to the main PR.
 - [ ] adds the `approved` label to the PRs, and sends out the Ballot Summary per this [template](#ballot-summary-email-template) to the [public Jakarta EE Specification Committee email list](jakarta.ee-spec@eclipse.org)
 - [ ] calculates the staged EFTL TCK signature and promotes it to the committee download area
  using the https://ci.eclipse.org/jakartaee-spec-committee/job/promote-release/ job. Manually editing the jenkins Build Information will help identify the build (ie. Mail 2.0 or CDI 3.0).
 - [ ] merges the specification (and apidocs) PRs, ensuring the "date:" field in the _index.md file has an appropriate value to allow publishing.
 - [ ] updates the specification page with the ballot results. This is normally done via a separate PR that should be reviewed, approved, and merged.
 - [ ] notifies the EMO of the ballot results by email to [emo@eclipse-foundation.org](emo@eclipse-foundation.org). Just forward the ballot summary note sent earlier to the public Spec Committee email list.
 - [ ] creates an issue in the specification project that includes the following checklist for the specification project team:
 \<include url of Issue here>

**Pull request merged**